commown_devices/models/common.py

```diff
@@ -81,6 +81,75 @@ def find_products_orig_location(env, products, stocks=None, compute_summary=Fals
     return {"pts_orig": pts_orig, "text_summary": location_summary}
 
 
+def create_move_from_lots(picking, located_lots):
+    """Create a stock move each lot. Do not réuse existing move because we might want to
```

Comment:
Typos dans la docstring:
- "move each lot" > "move for each lot" je suppose
- "réuse" > "reuse"

Reply:
Fait